package.json

```diff
@@ -108,7 +108,7 @@
 	"repository": "forcedotcom/sfdx-scanner",
 	"scripts": {
 		"build": "./gradlew build",
-		"prepack": "rm -rf lib && tsc -b && oclif-dev manifest && oclif-dev readme",
+		"prepack": "rm -rf lib && npx tsc -b && npx oclif-dev manifest && npx oclif-dev readme",
```

Comment:
Why is this needed?

Reply:
I sincerely have no idea. For some reason, when I ran the scripts on Linux, they worked fine, but only worked on Windows when I used `npx` to run the local versions instead of a global version.
I had to add `npx` to the `test` task when we first added the Windows tests, for the same reason.

Reply:
We should be able to remove this since we are no longer building on Windows.

Reply:
Good point. I'll do that.